src/components/SearchBar.tsx

```diff
@@ -1,23 +1,61 @@
-import { SearchField, Box, Label } from "./util"
+import { SearchField, Box, Label, StyledP, SearchResultButton, SearchResultSpan } from "./util"
 import styled from "styled-components"
-import { useEffect, useRef, useState } from "react"
+import React, { useEffect, useRef, useState } from "react"
```

Comment:
Could we maybe import `Fragment` as a named export instead, so we don't need to import everything from React?